packages/admin/admin/src/appHeader/dropdown/AppHeaderDropdown.tsx

```diff
@@ -68,41 +85,43 @@ function Dropdown({
     }, []);
 
     return (
-        <div className={classes.root} ref={rootRef}>
+        <Root ref={rootRef} {...slotProps?.root}>
             <AppHeaderButton endIcon={dropdownArrow !== null ? dropdownArrow(_open) : undefined} {...restProps} onClick={() => _onOpenChange(true)}>
                 {buttonChildren}
             </AppHeaderButton>
-            <Popover
-                classes={{ root: classes.popoverRoot, paper: classes.popoverPaper }}
-                style={{ "--comet-admin-app-header-dropdown-item-width": `${itemWidth}px` } as React.CSSProperties}
+            <PopoverRoot
+                {...slotProps?.popoverRoot}
                 open={_open}
                 anchorEl={rootRef.current}
                 anchorOrigin={{ vertical: "bottom", horizontal: "center" }}
                 transformOrigin={{ vertical: "top", horizontal: "center" }}
                 onClose={() => _onOpenChange(false)}
                 marginThreshold={0}
+                PaperProps={{
+                    sx: {
+                        minWidth: itemWidth,
+                    },
+                }}
                 {...popoverProps}
             >
                 {typeof children === "function" ? children(() => _onOpenChange(false)) : children}
-            </Popover>
-        </div>
+            </PopoverRoot>
+        </Root>
     );
 }
 
-export const AppHeaderDropdown = withStyles(styles, { name: "CometAdminAppHeaderDropdown" })(Dropdown);
-
 declare module "@mui/material/styles" {
-    interface ComponentNameToClassKey {
-        CometAdminAppHeaderDropdown: AppHeaderDropdownClassKey;
-    }
-
     interface ComponentsPropsList {
         CometAdminAppHeaderDropdown: Partial<AppHeaderDropdownProps>;
     }
 
+    interface ComponentNameToClassKey {
+        CometAdminAppHeaderDropdown: AppHeaderDropdownClassKey;
+    }
+
     interface Components {
         CometAdminAppHeaderDropdown?: {
-            defaultProps?: ComponentsPropsList["CometAdminAppHeaderDropdown"];
+            defaultProps?: Partial<ComponentsPropsList["CometAdminAppHeaderDropdown"]>;
```

Comment:
Only one of the prop-types (`ComponentsPropsList` or `Components`)  needs to be `Partial`, as one is used by the other.